### PR TITLE
Added a skip question button option for each of the question fragments 

### PR DESCRIPTION
**Pull request by rachelye1**

### CODE REVIEW

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentActivity.kt [hunk 1]
```diff
@@ -3,6 +3,9 @@ package org.sagebionetworks.assessmentmodel.presentation
 import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
+import org.sagebionetworks.assessmentmodel.Step
+import androidx.fragment.app.Fragment
+import android.widget.Button
```
Comment (nategbrown9) [inline presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentActivity.kt:8]: I don't think these imports are used.

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt [hunk 2]
```diff
@@ -14,9 +14,11 @@ import org.sagebionetworks.assessmentmodel.InstructionStep
 import org.sagebionetworks.assessmentmodel.Step
 import org.sagebionetworks.assessmentmodel.navigation.BranchNodeState
 import org.sagebionetworks.assessmentmodel.navigation.NavigationPoint
+import org.sagebionetworks.assessmentmodel.serialization.SkipCheckboxInputItemObject
 import org.sagebionetworks.assessmentmodel.survey.ChoiceQuestion
 import org.sagebionetworks.assessmentmodel.survey.MultipleInputQuestion
 import org.sagebionetworks.assessmentmodel.survey.SimpleQuestion
+import org.sagebionetworks.assessmentmodel.survey.SkipCheckboxInputItem
```
Comment (nategbrown9) [inline presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt:21]: Unused imports can be removed.

**Review by nategbrown9 — CHANGES_REQUESTED** [review]
Looks good, just a few unused imports to remove.  The next step would be to move the skip button into the StepNavigationBar so that it could be reused across steps without duplicating code.

**Review by nategbrown9 — COMMENTED** [review]
Looking good. Just a few changes:

- The hide and show code for the skip button that is duplicated in MultipleInputQuestionStepFragment and TextQuestionStepFragment can be moved into the setup method of StepNavigationBar. Can also handle hiding it for non-question steps here.
-  Check the step's buttonMap to see if there is a custom string to use for the button title.
-  Check the step's hideButton list to see if the skip button should be hidden.

File: presentation/src/main/res/layout/text_question_step_fragment.xml [hunk 3]
```diff
@@ -16,12 +17,28 @@
...
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_below="@id/question_header"/>
+        android:layout_below="@id/question_header">
+
+    </org.sagebionetworks.assessmentmodel.presentation.inputs.TextInputView>
 
-    <org.sagebionetworks.assessmentmodel.presentation.views.StepNavigationBar
-        android:id="@+id/nav_bar"
+    <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (liujoshua) [inline presentation/src/main/res/layout/text_question_step_fragment.xml:24]: Is this ConstraintLayout doing anything?

Comment (rachelye1) [reply presentation/src/main/res/layout/text_question_step_fragment.xml:24]: Yeah it is constraining the bar to the bottom of the screen.

**Pull request merged**